docs/source/model_design/designing_vivarium_model/interactive_sim/index.rst

```diff
@@ -296,8 +296,43 @@ choose to make plots in the interactive sim instead!
 Common Challenges
 -----------------
 
-.. todo::
+Using the interactive sim is fundamentally different than looking at simulation outputs 
+and this can lead to challenges. Because you are running a mini simulation on your computer, 
+you overlap a lot more with engineering workflow and have to watch out for some common pitfalls.
+
+1. Using the correct branch and simulation version: 
+
+Be sure that you are pulling from the engineering repo as needed. If you forget, the latest work 
+might not be present in your version of the sim, leading to confusion. 
+
+If you need to look at an old version of the sim or are actively debugging something, you might 
+not work from the main branch. Work with engineering to ensure you're on the correct branch for 
+your needed task. 
+
+This can be especially challenging if you need to use different branchs of upstream repos, like 
+vivarium or vivarium public health. If you run into this type of situation, consult with the 
+engineers to find the best strategy to move forward. 
+
+2. Editing engineering files on your local machine: 
```

Comment:
Just curious: are you usually doing this literally on your *local* machine, i.e. your laptop? I'm usually doing it on the cluster. But I think the wording here is still fine either way.

Reply:
It is on the cluster! I guess I mean that I open a file on my laptop and change it, but it is on the cluster